### ISSUE TITLE
ProcessID as Varibale leads to no longer editable process

### ISSUE THREAD

**Issue by bjoernb94** [camunda/camunda-modeler#1622]
__Describe the Bug__

If you use a variable in the format ${id} in the ProcessID field, the Camunda Modeler displays various warnings. The diagram can then no longer be viewed. If you specify the value, it will be displayed as valid. 

![Camunda-Modeler-3 5 0-Error](https://user-images.githubusercontent.com/9151364/70045578-a0934200-15c4-11ea-95ae-b2855e43ee9b.gif)

__Steps to Reproduce__

1. New BPMN Diagram
2. Create Pool/Participant
3. Change Process Id to ${id}
4. Save, close and open Diagram

__Expected Behavior__

It should be possible to set a variable in the format ${id} in the field ProcessID to be able to control it from the application. With this variable it should still be possible to edit the process in the Camunda Modeler.
In version 1.16.2 the use of variables as ProcessID still worked. 

__Environment__

- OS: Windows 10, Version: 1709
- Camunda Modeler Version: 3.5.0

**Comment by nikku** [comment]
Thanks for filling this bug report.

It looks like you stumble over enhanced validation we've shipped with [`v3.2.0`](https://github.com/camunda/camunda-modeler/blob/develop/CHANGELOG.md#320) of the app.

Starting with `v3.2.0` we require that XML documents (BPMN / CMMN / DMN) are valid when it comes to IDs. In the past it was possible to inject illegal (according to the XML standard) characters into the ID. Now the modeler refuses to import such documents.

Why was this change necessary? Because we've identified that injecting arbitrary characters into XML IDs (user input) is a security hole that may lead to HTML injection in any place where the BPMN diagram is potentially embedded (Cockpit, ...). 

It is a bug that we still allow `${FOO}` to be entered as an ID in the properties panel: https://github.com/camunda/camunda-modeler/issues/1623.
